test/acceptance/tests.py

```diff
@@ -15,8 +16,10 @@
     SubmissionPage, AssessmentPage, GradePage, StaffAreaPage
 )
 
+# This value is generally used in jenkins, but not locally
+PROFILING_ENABLED = os.environ.get('ORA_PROFILING_ENABLED', False)
 
-def retry(tries=4, delay=3, backoff=2):
+def retry(tries=2, delay=4, backoff=2):
```

Comment:
Oh yeah, good point, that definitely sounds do-able. I'll write up a ticket to have the team discuss that possibility.